package.nls.json

```
@@ -108,7 +108,8 @@
     "AWS.command.refreshAwsExplorer": "Refresh Explorer",
     "AWS.command.refreshCdkExplorer": "Refresh CDK Explorer",
     "AWS.command.cdk.help": "View CDK Documentation",
-    "AWS.command.ec2.connectToInstance": "Connect to EC2 Instance...",
+    "AWS.command.ec2.openTerminal": "Open terminal in EC2 instance...",
```

Wording feels clunky. Maybe this is better?
```suggestion
    "AWS.command.ec2.openTerminal": "Open terminal to EC2 instance...",
```
Could also use "for" instead of "to"